deployments/kustomize/overlays/local/phpmyadmin/deployment.yaml

```
@@ -0,0 +1,30 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: phpmyadmin
+  labels:
+    app.kubernetes.io/version:	11.5.2
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: phpmyadmin
+  template:
+    metadata:
+      labels:
+        app: phpmyadmin
+    spec:
+      containers:
+        - name: phpmyadmin
+          image: phpmyadmin/phpmyadmin
```

```suggestion
          image: phpmyadmin/phpmyadmin:5.2
```
i think it is always good to put a tagged version, otherwise somwhere down the line this could stop working, because of a major release... it also indicates at least a ball park version